src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -624,7 +648,13 @@ def __lnext(self):
             if not line:
                 raise StopIteration
             fields = line.split(None, 8)
-            if len(fields) < 6 or len(fields[0]) != 17 or fields[0][2] != '/':
+            if (
+                # pylint: disable=too-many-boolean-expressions
+                len(fields) < 7
+                or len(fields[0]) != 10 or fields[0][4] != '/' or fields[0][7] != '/'
+                or len(fields[1]) != 15 or fields[1][2] != ':' and fields[1][8] != '.'
+                # pylint: enable=too-many-boolean-expressions
+            ):
```

Comment:
You have three copies of this. Can we transform it into a helper function or a static method of the `LogLine` class? Please consider.